assets/js/main.js

```diff
@@ -3,55 +3,67 @@
 /* eslint-disable space-in-parens */
 /* eslint-disable no-trailing-spaces */
 /* eslint-disable prefer-template */
+/* eslint-disable class-methods-use-this */
```

Comment:
- [ ] Please don't disable linters because they help to enforce a strict coding style and ensure that your codebase looks the same. Try to fix your code to adhere to the linter rules rather than outrightly disabling them 🙏